packages/rn-tester/android/app/src/main/jni/Android.mk

```diff
@@ -15,9 +15,10 @@ LOCAL_PATH := $(THIS_DIR)
 
 include $(CLEAR_VARS)
 LOCAL_MODULE := rntester_appmodules
-# Note: Build the react-native-codegen output along with other app-specific C++ files.
+# Note: We are linking against react_codegen_rntester hence no need to built the react-native-codegen output.
 LOCAL_C_INCLUDES := $(LOCAL_PATH) $(GENERATED_SRC_DIR)/codegen/jni
-LOCAL_SRC_FILES := $(wildcard $(LOCAL_PATH)/*.cpp) $(wildcard $(GENERATED_SRC_DIR)/codegen/jni/*.cpp)
+LOCAL_SRC_FILES := $(wildcard $(LOCAL_PATH)/*.cpp)
+LOCAL_SRC_FILES := $(subst $(LOCAL_PATH)/,,$(LOCAL_SRC_FILES))
```

Comment:
This needs to be changed to:

```
LOCAL_C_INCLUDES := $(LOCAL_PATH) $(GENERATED_SRC_DIR)/codegen/jni
LOCAL_SRC_FILES := $(wildcard $(LOCAL_PATH)/*.cpp) $(wildcard $(GENERATED_SRC_DIR)/codegen/jni/*.cpp)
LOCAL_SRC_FILES := $(subst $(LOCAL_PATH)/,,$(LOCAL_SRC_FILES))
```

Reply:
@cortinico Do you mean to revert this change in line #20 ? 
If so, I'm not getting why we need to compile the codegen sources in this module.. They are compiled into react_codegen_rntester.so which is being linked into this module. 
Moreover, our fix to convert the absolute paths into relative path by stripping the local path don;t work for files outside the current directory!

Reply:
Yup you're right 👍  Closing this comment. Thanks for explaining it.